src/components/lib/c_stub.c

```diff
@@ -0,0 +1,11 @@
+
+#include <cos_types.h>
+#include <cstub.h>
+
+__attribute__((regparm(1))) int
```

Comment:
clever use of regparm here to capture the dynamically generated stub's behavior (from `gen_client_stubs.c`).